### DISCUSSION TITLE
Option to skip a day for numerical habits as well

### DISCUSSION THREAD

**Discussion by KristianTashkov** [iSoron/uhabits#1123] [Feature Requests] [answered]
Similarly to YES/NO habits, some days it's just impossible to do a habit (you are sick, something outside of your control makes it impossible to complete, etc). This isn't connected to the type of habit though, and even for a "Run X miles every week", you might want to skip a week that you were enable to go running, and not see your well earned score decrease during that time.

**Comment by iSoron** [comment, marked as answer]
This feature will become available in Loop 2.1.
